### DISCUSSION TITLE
10,000 contributions

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [EddieHubCommunity/support#1637] [General]
I hit 9,999 contributions this morning.

I thought to myself - the best way to hit that 10,000th would be with a contribution to EddieHub. So here's a discussion!

Tell me about your contribution goals. Are there any projects you dream of contributing to? How can we help you reach your goals?

**Comment by schmelto** [comment]
@nhcarrigan you are the best ! 🚀

**Reply by eddiejaoude** [reply]
I agree 💥

**Comment by eddiejaoude** [comment]
EddieHub projects are my favourite, but I might be a bit biased 😂

**Reply by naomi-lgbt** [reply]
They *are* pretty fun to contribute to 😉

**Comment by kaiwalyakoparkar** [comment]
- Almost every EddieHub Project
- FreeCodeCamp Projects
- CNCF projects
- GSOC orgs (In general good open-source orgs. GSOC brings them together so formed it that way)
